packages/editor-sdk/src/components/Widgets/StyleWidgets/FontWidget.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { HStack, Select, Text, VStack } from '@chakra-ui/react';
+import { CORE_VERSION, StyleWidgetName } from '@sunmao-ui/shared';
+import { WidgetProps } from '../../../types/widget';
+import { implementWidget } from '../../../utils/widget';
+import { ExpressionEditor } from '../../Form';
+
+type Font = {
+  fontSize?: string | number;
+  fontWeight?: string | number;
+  textAlign?: string;
+};
+
+const WeightOptions = [
+  { value: 100, label: 'Thin' },
+  { value: 200, label: 'Extra Light' },
+  { value: 300, label: 'Light' },
+  { value: 400, label: 'Normal' },
+  { value: 500, label: 'Medium' },
+  { value: 600, label: 'Semi Bold' },
+  { value: 700, label: 'Bold' },
+  { value: 800, label: 'Extra Bold' },
+  { value: 900, label: 'Black ' },
+];
+const AlignOptions = [
+  { value: 'left', label: 'Left' },
+  { value: 'center', label: 'Center' },
+  { value: 'right', label: 'Right' },
+];
+
+export const FontWidget: React.FC<WidgetProps<{}, Font>> = props => {
+  const { value, onChange } = props;
+
+  return (
+    <VStack width="full">
+      <HStack width="full">
+        <Text>Size</Text>
+        <ExpressionEditor
```

Comment:
It could use the `ExpressionWidget` to be replaced because the `ExpressionWidget` would evaluate and display the expression value and errors.